classes/item.rb

```diff
@@ -0,0 +1,41 @@
+require 'date'
+
+class Item
+  attr_accessor :genres, :authors, :labels, :source, :publish_date, :archived
+  attr_reader :id
```

Comment:
- [ ] I think `genres, authors, labels` supposed to use the getters method and not setters. ℹ️ A getter method allows us to access the value of an instance variable, `genres, authors and labels` in this example.